### PR TITLE
21 create events details page and enable viewing of event details announcements

### PR DESCRIPTION
**Pull request by Sundance636**
I tested viewing event details using the dummy events in the database, once we get the QR stuff working, we can easily extend functionality to any event whose QR you input in the 'fetcheventdetails()' method

### CODE REVIEW

**Review by Logan9312 — APPROVED** [review]
Looks well documented and no obvious issues

File: app/src/main/java/com/example/scanpal/MainActivity.java [hunk 1]
```diff
@@ -1,8 +1,12 @@
 package com.example.scanpal;
 
 import android.os.Bundle;
+import android.view.ViewGroup;
```
Comment (Nandan-18) [inline app/src/main/java/com/example/scanpal/MainActivity.java:4]: are these imports required?

File: app/src/main/java/com/example/scanpal/EventDetailsFragment.java [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+        View view = inflater.inflate(R.layout.event_details, null, false);
+
+        //retrieves all the info about specific event from database
+        fetchEventDetails();
+
+        Log.d("fire", "end of onCreate");
+
+        return view;
+    }
+
```
Comment (Nandan-18) [inline app/src/main/java/com/example/scanpal/EventDetailsFragment.java:54]: you might want to add navigation for the back button to go back to the events page

**Review by Nandan-18 — APPROVED** [review]
lgtm! can be merged once comments are resolved!

File: app/src/main/java/com/example/scanpal/MainActivity.java (hunk 1 above)
Comment (Sundance636) [reply app/src/main/java/com/example/scanpal/MainActivity.java:4]: Nope! ill get rid of that, it was a remnant from when I was testing something else.

File: app/src/main/java/com/example/scanpal/EventDetailsFragment.java (hunk 2 above)
Comment (Sundance636) [reply app/src/main/java/com/example/scanpal/EventDetailsFragment.java:54]: Got it!

**Review by Nandan-18 — APPROVED** [review]
looks good

**Pull request merged**